### PR TITLE
Feature/dockerize business

### PR DESCRIPTION
**Pull request by ptitdam2001**
- dockerize business API
- fix devlopment reverse proxy with business

Please test the frontend interface, you should see a SeriesList and a StudiesList

### CODE REVIEW

File: business/tests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using Xunit;
+
+namespace tests
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void Test1()
+        {
+
```
Comment (cquennouelle) [inline business/tests/UnitTest1.cs:11]: Amazing unit test coverage ;)

File: client/web/src/components/series/list/SeriesList.tsx [hunk 2]
```diff
@@ -1,7 +1,24 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
```
Comment (bat3) [inline client/web/src/components/series/list/SeriesList.tsx:2]: Why use axios ?
It is more a question for general culture 🤪

**Review by cquennouelle — APPROVED** [review]

File: business/tests/UnitTest1.cs (hunk 1 above)
Comment (ptitdam2001) [reply business/tests/UnitTest1.cs:11]: lol a boilerplate ;)

**Review by bat3 — APPROVED** [review]

File: client/web/src/components/series/list/SeriesList.tsx (hunk 2 above)
Comment (ptitdam2001) [reply client/web/src/components/series/list/SeriesList.tsx:2]: It is a REST client in JS and there is a testing lib for axios

**Pull request merged**